index.html

```diff
@@ -151,23 +151,29 @@ <h2>Survey</h2>
                 <form   
                 action="https://formspree.io/xjvkplpr"
                 method="POST">
-                    <label>What is your name?<input type="text" name="name_field"></label>
-                    <label>What is your email?<input type="email" name="email_field"></label>
-                    <label>Which Integer Is best?<input type="number" name="best_integer"></label>
-                    <label>Which day is best?<input type="date" name="best_day"></label>
-                    <label>Which bear is best?<select type="select" name="best_bear">
+                    <label for="name_field">What is your name?</label>
+                    <input type="text" name="name_field" id="name_field" required>
+                    <label for="email_field">What is your email?</label>
+                    <input type="email" name="email_field" id="email_field" required>
+                    <label for="best_integer">Which Integer Is best?</label>
+                    <input type="number" name="best_integer" id="best_integer">
+                    <label for="best_day">Which day is best?</label>
+                    <input type="date" name="best_day" id="best_day">
+                    <label for="best_bear">Which bear is best?</label>
+                    <select type="select" name="best_bear" id="best_bear">
```

Comment:
The lab asked you to use the 'for' and 'id' technique for associating labels with controls